### PR TITLE
Update footer links and labels

### PR DESCRIPTION
**Pull request by pabloest**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

**Comment by pabloest** [comment]
### Preview

theCleanShot-2023-02-13 at 20 18 18

File: docusaurus.config.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ module.exports = {
           ],
         },
       ],
-      copyright: `Copyright © ${new Date().getFullYear()} r2c. Semgrep®️  is a registered trademark of r2c. This site is built with Docusaurus.`,
+      copyright: `Copyright © ${new Date().getFullYear()} r2c. Semgrep®️  is a registered trademark of r2c. These docs is made with Docusaurus.`,
```
Comment (s-santillan) [inline docusaurus.config.js:130]:
```suggestion
      copyright: `Copyright © ${new Date().getFullYear()} r2c. Semgrep®️  is a registered trademark of r2c. These docs are made with Docusaurus.`,
```

**Review by s-santillan — APPROVED** [review]
lgtm, only 1 comment!

**Comment by pabloest** [comment]
Thank you, @s-santillan.

**Pull request merged**